Dockerfile

```diff
@@ -1,8 +1,10 @@
 FROM nginx:stable
-RUN apt-get -y update
-RUN apt-get -y install git
+RUN apt-get -y update && apt-get -y install git
 WORKDIR /usr/src
-RUN git clone https://github.com/Ada18980/KinkyDungeonStandalone.git .
+RUN git clone -b PickSrc https://github.com/Demopans/KinkyDungeonStandalone.git .
```

Comment:
I think this should point to my git and not yours o.o

Reply:
I thought I reverted that...

Reply:
Oh wait, I was doing that so stuff builds on my end